.github/workflows/ci.yml

```diff
@@ -0,0 +1,87 @@
+# SPDX-FileCopyrightText: 2022 Helmholtz Centre for Environmental Research (UFZ)
+# SPDX-FileCopyrightText: 2022 Helmholtz-Zentrum Dresden-Rossendorf (HZDR)
+#
+# SPDX-License-Identifier: Apache-2.0
+
+---
+name: CI
+on:
+  pull_request:
+  push:
+    branches:
+      - "main"
+    tags:
+      - "v*.*.*"
+
+env:
+  PY_COLORS: 1
+  ANSIBLE_FORCE_COLOR: 1
+  WORKON_HOME: ~/.cache/venv
+  PIPENV_CACHE_DIR: ~/.cache/pipenv
+  PIP_CACHE_DIR: ~/.cache/pip
```

Comment:
I just learned about the caching feature of the `setup-python` action and suggest to use this instead.

```suggestion
```